docs/compatibility.md

```diff
@@ -95,7 +95,7 @@ Distrobox guests tested successfully with the following container images:
 | Ubuntu | 14.04<br>16.04<br>18.04<br>20.04<br>21.10<br>22.04 | docker.io/library/ubuntu:14.04<br>docker.io/library/ubuntu:16.04<br>docker.io/library/ubuntu:18.04<br>docker.io/library/ubuntu:20.04<br>docker.io/library/ubuntu:21.10<br>docker.io/library/ubuntu:22.04    |
 | Kali Linux | rolling | docker.io/kalilinux/kali-rolling:latest |
 | Void Linux | | ghcr.io/void-linux/void-linux:latest-thin-bb-x86_64 <br> ghcr.io/void-linux/void-linux:latest-thin-bb-x86_64-musl <br> ghcr.io/void-linux/void-linux:latest-full-x86_64 <br> ghcr.io/void-linux/void-linux:latest-full-x86_64-musl |
-
+| Gentoo Linux | rolling | [DOCKERFILE HERE](https://github.com/89luca89/distrobox/issues/112#issue-1098097949)
```

Comment:
Would be better to really explain here, let's point to the official resource here: https://github.com/gentoo/gentoo-docker-images#using-the-portage-container-in-a-multi-stage-build

and maybe provide the example and specify to the user that they need to create the image themselves and it's not ready to use on a registry

Reply:
I'll create a file and link to it.